GDCpp/GDCpp/Extensions/Builtin/MouseTools.cpp

```diff
@@ -76,6 +76,14 @@ int GD_API GetMouseWheelDelta(RuntimeScene &scene) {
   return scene.GetInputManager().GetMouseWheelDelta();
 }
 
+bool GD_API ScrollUp(RuntimeScene &scene) {

```

Comment:
Use a long, precise name for these "free functions":
`IsMouseWheelScrollingUp`, `IsMouseWheelScrollingDown`

Reply:
The "Is" is important, otherwise it looks like a function that does the action of scrolling up or down, instead of returning if scrolling is being done or not.